<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-FVC0VK09" ignored="1" name="prepay_login" tests="9" failures="1" timestamp="2021-05-05T22:26:31 IST" time="0.079" errors="0">
  <testcase name="address_change" time="0.009" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="test1" time="0.011" classname="testNG_Tutorial.testNG_Basic">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG_Tutorial.testNG_Basic.test1(testNG_Basic.java:18)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="after_suite" time="0.005" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="name_change" time="0.002" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="personal_details_DOB_change" time="0.001" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="personal_details_email_change" time="0.002" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="personal_details_number_change" time="0.001" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="personal_details_number_change" time="0.002" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="personal_details_number_change" time="0.002" classname="testNG_Tutorial.testNG_2"/>
  <testcase name="test2" time="0.0" classname="testNG_Tutorial.testNG_Basic">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- prepay_login -->
